### ISSUE TITLE
adding new user in one browser shows up multiple times in other browser user list

### ISSUE THREAD

**Issue by benjamin79** [ryanswapp/react-phoenix-starter-template#10]
Thank you for making this.

Using your boilerplate without any changes leads to the following behavior when doing the following steps:
- Open user list in one browser
- Open signup in another browser - Add new user 
  -> Depending on how many users there are, a new user shows up as many times as there are users in the user list

For example:If there are already 4 users in the db and you add another one, the user list shows the new user 4 times. Clicking to another route like home and back the user list everything is normal.

I have to add that i´m new to phoenix.

thank you

**Comment by ryanswapp** [comment]
Interesting. Thanks for reporting. I'll take a look and see what's going on.

**Comment by samcorcos** [comment]
Yeah, this happened to me too. But for some reason, it doesn't happen every time.
